### PR TITLE
Update process to analyse and manage open trade in priority

### PR DESCRIPTION
**Pull request by Axel-CH**
## Summary
This feature allow freqtrade, at each process loop to manage in priority open trades when "adjust_trade_position" is enabled. With this feature, the bot will not wait the full whitelist analysis to manage open trades. This feature is usefull when there is a substantial quantity of pairs in whitelist and user need to quickly manage open trades after a candle close.

With this feature I was able to divide the trade management lag by 3.

## Quick changelog

- add new method manage_open_trades to prevent duplicate code in process loop
- manage first open trades when position adjustment is enable

## What's new?
This PR improve position trade management time for open position live

### CODE REVIEW

File: freqtrade/freqtradebot.py [hunk 1]
```diff
@@ -225,25 +225,23 @@ def process(self) -> None:
...
-        # Check if we need to adjust our current positions before attempting to buy new trades.
-        if self.strategy.position_adjustment_enable:
+            # Check if we need to adjust our current positions before attempting to buy new trades.
             with self._exit_lock:
                 self.process_open_trade_positions()
 
+        remaining_pairs_to_analyse = [pair for pair in self.active_pair_whitelist if pair not in open_trade_pairs_analyzed]
+
+        self.strategy.analyze(remaining_pairs_to_analyse)
+        self.manage_open_trades()
```
Comment (xmatthias) [inline freqtrade/freqtradebot.py:243]: Seems like you're calling `manage_open_trades` twice per iteration - which we clearly wouldn't want, as that will call all callbacks twice per loop - potentially altering behavior in some odd strategies.

**Review by xmatthias — COMMENTED** [review]
> With this feature I was able to divide the trade management lag by 3.

You should also note that it's at the expense of delaying entry to potential new trades by a much higher factor (which i hope you can provide based on your tests).

File: freqtrade/freqtradebot.py (hunk 1 above)
Comment (Axel-CH) [reply freqtrade/freqtradebot.py:243]: Yes it is the case, do you think we could manage_open_trade, except the ones that was already managed. As the goal of this feature is to manage in priority open trades, it think two call is required. If you think it is not worth it I can scrap this PR and focus on other way to speed up live trades management.

Comment (xmatthias) [reply freqtrade/freqtradebot.py:243]: I don't understand the need for this call.

there is no need for this in my eyes, it'll handle the SAME trades again (there can't have been new trades opened between the first and 2nd call (except through force-entry - where it won't matter if they're handled the next iteration).

**Comment by Axel-CH** [comment]
After successful experiments, i'm scrapping this in favor of dataframe multiprocessing custom indicators populate and merging. With the gained speed introduced by parallelization, prioritizing trade entry or management is no longer relevant.

**Pull request closed without merging**